src/kernel/Cargo.toml

```diff
@@ -28,8 +28,15 @@ addr2line = { version = "0.16.0", default-features = false, features = [
 backtracer_core = { git = "https://github.com/twizzler-operating-system/backtracer", branch = "twizzler" }
 limine = "0.1.11"
 twizzler-queue-raw = { version = "*", path = "../lib/twizzler-queue-raw", default-features = false }
-syscall_encode = { version = "0.1.2" }
+# syscall_encode = { version = "0.1.2" }
 volatile = "0.5"
+uart_16550 = "0.3.0"
+x86 = "0.52.0"
```

Comment:
Some of these are target dependent -- do they have to be moved back up here?

Reply:
It does create an error just removing them:
use of undeclared crate or module `x86`

I can try and find out if there is a way to either
A) Make Kani recognize target-dependent dependencies or
B) Have a seperate category for Kani complication

Reply:
Should be fixed by [ee3a25d](https://github.com/twizzler-operating-system/twizzler/pull/210/commits/ee3a25d27c5b34677b963f581e6ca2fc34249b37)